### PR TITLE
First version of serviceitems, with syncronous

### PR DESCRIPTION
**Pull request by TVNielsen**
Changes:
- The StripePlan in a StripeSubscription can now be null if there are multiple SubscriptionItems included 
- StripeSubscription now supports SubscriptionItems, a collection of individual plans in a subscription instead of having only one.
- StripeSubscriptonItem is added, to encompass a StripePlan with a quantity on a subscription
-  Infrastructure.Urls now contain SubscriptionItem base URL
- The SubscriptionItemService is added to the Subscription namespace

### CODE REVIEW

File: src/Stripe.net/Entities/StripeSubscriptionItem.cs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+﻿using Newtonsoft.Json;
```
Comment (jaymedavis) [inline src/Stripe.net/Entities/StripeSubscriptionItem.cs]: this file is already in the library

File: src/Stripe.net/Infrastructure/Urls.cs [hunk 2]
```diff
@@ -36,6 +36,8 @@ internal static class Urls
 
         public static string Subscriptions => BaseUrl + "/subscriptions";
 
+        public static string SubscriptionItems => BaseUrl + "/subscription_items";
+
```
Comment (jaymedavis) [inline src/Stripe.net/Infrastructure/Urls.cs]: I am working towards eliminating this file. The urls are going directly in the service - see the StripeSubscriptionItemService

**Comment by jaymedavis** [comment]
Sorry if this was confusing the other day - I have already implemented this feature. The only thing I seem to be missing from here is the link to subscription items on StripeSubscription

File: src/Stripe.net/Infrastructure/Urls.cs (hunk 2 above)
Comment (TVNielsen) [inline src/Stripe.net/Infrastructure/Urls.cs]: Ok, yea great idea, one place less to maintain :)

**Pull request closed without merging**